src/components/EmptyState/EmptyState.tsx

```diff
@@ -69,6 +72,14 @@ export default class EmptyState extends React.PureComponent<Props, never> {
       ? buttonFrom(secondaryAction, {plain: true})
       : null;
 
+    const footerMarkup = footer ? (
+      <div className={styles.Footer}>
+        <TextContainer>
+          <div className={styles.FooterContent}>{footer}</div>
```

Comment:
I think we can get rid of this extra div by applying the subdued directly to `Footer`:

```scss
.Footer {
  @include text-emphasis-subdued;
}
```